src/components/performance/views.tsx

```diff
@@ -0,0 +1,200 @@
+    metric: props.metric,
+    units: props.units,
+  });
+
+  return (
+    <div className="govuk-grid-row govuk-!-padding-bottom-9">
+      <div className="govuk-grid-column-full">
+        <h2 className="govuk-heading-m" id={props.id}>
+          {props.title}
+        </h2>
+      </div>
+
+      <div className="govuk-grid-column-full">
+        <p className="govuk-body">{props.description}</p>
+
+        <MetricChart chart={props.chart.outerHTML} />
+
+        {props.children}
+
+        <div className="govuk-grid-column-full">
+          <a href={downloadLink.toString()} className="govuk-link" download>
+            Download &quot;{props.titleText || props.title}&quot; as a CSV
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export function MetricPage(props: IMetricPageProperties): ReactElement {
+  return (
+    <div className="govuk-width-container">
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds-from-desktop">
+          <h1 className="govuk-heading-xl">
+            <span className="govuk-caption-xl">GOV.UK Platform as a Service</span> {}
+            Performance dashboard
+          </h1>
+          <p className="govuk-body">
+            GOV.UK PaaS is a shared platform that public sector service teams can use to quickly host their applications
+            in the cloud.
+          </p>
+
+          <p className="govuk-body">
+            Each point on the charts is aggregated data over {}
+            <strong className="non-breaking">{props.period.humanize()}</strong>.
+          </p>
+
+          <hr className="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+
+          <div className="govuk-grid-row govuk-!-padding-bottom-9">
+            <div className="govuk-grid-column-full">
+              <h2 className="govuk-heading-m" id="uptime">
+                Uptime
+              </h2>
+            </div>
+
+            <div className="govuk-grid-column-full">
+              <p className="govuk-body">
+                Uptime is measured as the availability of the GOV.UK PaaS API on the platform.
+              </p>
+            </div>
+
+            <div className="govuk-grid-column-one-half">
+              <p className="govuk-body">
+                <strong className="govuk-!-font-size-48">99.95%</strong><br />
+                <small className="govuk-!-font-size-27">uptime in the {props.region} region over the last year</small>
+              </p>
+            </div>
+          </div>
+
+          {props.organizations
+            ? <Metric
+                id="organisations"
+                format="number"
+                title="Number of organisations"
+                // eslint-disable-next-line max-len
+                description="The number of organisations, or orgs, represents a group of users, applications and environments. We have 2 types of organisations: 3 month trial accounts and billable accounts."
```

Comment:
The number of organisations represents a group of users, applications and environments. We have 2 types of organisation: 3 month trial accounts and billable accounts.